pulp_container/app/registry_api.py

```diff
@@ -1001,26 +985,58 @@ def put(self, request, path, pk=None):
                 tag = models.Tag.objects.get(name=tag.name, tagged_manifest=manifest)
                 tag.touch()
 
+            add_content_units += [str(tag.pk), str(manifest.pk)]
+
+            if is_manifest_list:
```

Comment:
please add here else branch for image manifest

Reply:
Yeah, it makes sense now to move things like "add_content_units += [str(blob.pk) for blob in blobs_qs] + [str(config_blob.pk)]" here then.